portal/models/user.py

```diff
@@ -113,7 +113,7 @@ def children(self):
         raise NotImplementedError
 
 
-def permanently_delete_user(username, user_id=None, acting_user=None):
+def permanently_delete_user(username, user_id=None, acting_user=None, actor=None):
```

Comment:
`acting_user` is a DB object, `actor` is a string

looks like we need to support both- string for Flask CLI, DB object for [creating accounts](https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/views/user.py#L240)